src/core/agent/services/signifyNotificationService.ts

```diff
@@ -417,7 +421,23 @@ class SignifyNotificationService extends AgentService {
               });
               break;
             }
-
+            case OperationPendingRecordType.Oobi: {
+              if (!pendingOperation.metadata?.connectionId) {
+                break;
+              }
+              const connectionRecord = await this.connectionStorage.findById(
+                pendingOperation.metadata?.connectionId
+              );
+              if (connectionRecord) {
+                connectionRecord.pending = false;
+                await this.connectionStorage.update(connectionRecord);
+              }
+              callback({
+                opType: pendingOperation.recordType,
+                oid: recordId,
```

Comment:
Hmm, I wanted to cover the blind OOBIs later but maybe we should do it now because the schema is changing with this PR for long running operations. And actually, this `oid` here would need to be the `connectionId`.

Can you test something for me with the help of @Sotatek-BaoHoanga (who developed the DB layer):
- Create a connection record with ID that is the same as the operation `recordId` here
- When updating the record, can you update the ID of the record to now be the connectionId? (this will be in the OOBI response once it's done like before in `operation.response.i`

Does this work?

Reply:
^ above is possible with some changes to the DB layer but may not be compatible with askar if/wen we bring that in. Resolving and we can cover blind OOBIs (and those without `/oobi` in the URL) later